### PR TITLE
Initial partial version of vineyardData

### PR DESCRIPTION
**Pull request by maticserc**

### CODE REVIEW

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
See the invidual comments on the files

File: SMARTAGRIFOOD/vineyardData/CONTRIBUTORS.YAML [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ organization: Elmibit / eVineyard
+ project: eV2W (i4Trust)
+ comments:
+-
+ name: NAMEOFCONTRIBUTOR
+ surname: SURNAMEOFCONTRIBUTOR
+ mail: MAILOFCONTRIBUTOR
+ organization: ORGOFCONTRIBUTOR
+ project:
+ comments:
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/CONTRIBUTORS.YAML]: Remove extra template values

File: SMARTAGRIFOOD/vineyardData/examples/example-normalized.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/examples/example-normalized.json:1]: there is no id in this payload, this is not a valid one 
the object AgriVineyardBlock with only one subproperty is a bit weird
'subproperties' ?? Why use these name for creating a two levels object

File: SMARTAGRIFOOD/vineyardData/notes.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+notesHeader:
+  Vineyard data model definition is currently in incubated stage (work in progress). It is incomplete and large changes may occur at this stage, so it is highly advisable to not use the model until the version 1.0 is reached.
+notesMiddle:
+  notes appearing in the middle of the spec
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/notes.yaml]: remove the templated values

File: SMARTAGRIFOOD/vineyardData/schema.yaml [hunk 4]
```diff
@@ -0,0 +1,192 @@
+AgriVineyardBlock:
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/schema.yaml]: remove file.

File: SMARTAGRIFOOD/vineyardData/schema.json [hunk 5]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/schema.json:1]: Much to comment on here please see a valid example [like this](https://github.com/smart-data-models/dataModel.Agrifood/blob/master/AgriFarm/schema.json) and the [contribution manual](https://bit.ly/contribution_manual) and try to make similarly 
1.- id attribute of the model is missing
2.- The x-ngsi attributes. Why are they here (are you mapping the automatically generated .yaml). Please don't include it. See contribution manual and the example.
3.- There are many attributes with are included in the [GSMA-Commons](https://github.com/smart-data-models/data-models/blob/master/common-schema.json) (you do  not need to list manually , just include the $ref like the example
4.- The attributes descriptions are not including Property/Relationship/Geopropery (see contribution manual pages 7-10)
5.- address is a very common and generic attribute,(is included in the  [Location-Commons](https://github.com/smart-data-models/data-models/blob/master/common-schema.json) by the $ref (see example) recode with another name
6.- There is not keyvalues payload to be validated by this schema.

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
The .YAML is lowercase

File: SMARTAGRIFOOD/vineyardData/schema.json (hunk 5 above)
Comment (maticserc) [reply SMARTAGRIFOOD/vineyardData/schema.json:1]: Thanks Alberto for the guidance. Yes, as you may figured, we started with YAML schema, and then transformed it into JSON, and then generated example data based on JSON schema. Some strange things appeared in the process, aparently.

A question on point 6 - is this referring to sample data that is provided?

File: SMARTAGRIFOOD/vineyardData/examples/example-normalized.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  "AgriVineyardBlock": {
```
Comment (albertoabellagarcia) [inline SMARTAGRIFOOD/vineyardData/examples/example-normalized.json]: This is not a payload from a NGSIv2 context broker in normalized format

File: SMARTAGRIFOOD/vineyardData/schema.json (hunk 5 above)
Comment (albertoabellagarcia) [reply SMARTAGRIFOOD/vineyardData/schema.json:1]: yes once you have a payload in the broker just make a -d keyvalues export and then you got the example.

This is the one validated with the schema.json

**Review by albertoabellagarcia — APPROVED** [review]
There is a missing file, the example.json to be validated by the schema.

**Pull request merged**